test/built-ins/DataView/prototype/setFloat32/index-to-integer.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2015 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es7id: 24.2.1.2
+description: >
+  The requested index is converted with ToInteger.
+info: >
+  SetViewValue ( view, requestIndex, isLittleEndian, type, value )
+
+  ...
+  3. Let numberIndex be ToNumber(requestIndex).
+  4. Let getIndex be ? ToInteger(numberIndex).
```

Comment:
Adding a test case with an object coerced to a valid integer would make it more complete.